templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url 'taxi:driver-detail' pk=user.id %}">User: {{ user.username }}</a></li>
```

Comment:
The URL for the driver detail view uses the `user.id` as the primary key. Ensure that the URL pattern `taxi:driver-detail` is correctly set up to accept a `pk` argument and that the `user` object has an `id` attribute that corresponds to a `Driver` instance.